pkgs/development/libraries/libhttpseverywhere/default.nix

```diff
@@ -1,24 +1,33 @@
-{stdenv, fetchurl, gnome3, glib, json_glib, libxml2, libarchive, libsoup, gobjectIntrospection, meson, ninja, pkgconfig,  valadoc}:
+{stdenv, fetchurl, gnome3, glib, json_glib, libarchive, libsoup, gobjectIntrospection, meson, ninja, pkgconfig,  valadoc}:
 
 stdenv.mkDerivation rec {
-  major = "0.4";
-  minor = "5";
+  major = "0.6";
+  minor = "1";
   version = "${major}.${minor}";
 
   name = "libhttpseverywhere-${version}";
 
   src = fetchurl {
     url = "mirror://gnome/sources/libhttpseverywhere/${major}/libhttpseverywhere-${version}.tar.xz";
-    sha256 = "07sgcw285rl9wqr5k7srs3fj7fhgrrw6w780jx8wy8jw2bfwlvj2";
+    sha256 = "0pwnh5mbbcb9hpx7by5f5j5qj7kchflwx7dzh49hbkw1jvwrx655";
   };
 
   nativeBuildInputs = [ gnome3.vala valadoc  gobjectIntrospection meson ninja pkgconfig ];
-  buildInputs = [ glib gnome3.libgee libxml2 json_glib libsoup libarchive ];
+  buildInputs = [ glib gnome3.libgee json_glib libsoup libarchive ];
+
+  mesonFlags = "-Denable_valadoc=true";
 
   doCheck = true;
 
   checkPhase = "./httpseverywhere_test";
 
+  outputs = [ "out" "devdoc" ];
+
+  fixupPhase = ''
```

Comment:
The files should be in the same relative path. You can replace the whole `fixupPhase` with `moveToOutput share/devhelp/books $devdoc` but it would be best to put it in the [setuphook](https://github.com/NixOS/nixpkgs/blob/6bbc3a0b245ccf392a8ddbce41b372c17409f0e9/pkgs/build-support/setup-hooks/multiple-outputs.sh#L137).

Reply:
how do I put it in there? I don't really get, how the setup-hook works yet.

Reply:
Just add the line to `pkgs/build-support/setup-hooks/multiple-outputs.sh`, specifically the `_multioutDocs` function. It should be called automatically when the `outputs` attribute is present.

Reply:
doesn't it [automatically remove `devdoc`](https://github.com/NixOS/nixpkgs/blob/6bbc3a0b245ccf392a8ddbce41b372c17409f0e9/pkgs/build-support/setup-hooks/multiple-outputs.sh#L41)

Reply:
If I understand it correctly, the REMOVE case is for when `outputs` does not contain `devdoc`.

Reply:
Right.  `_overrideFirst outputDevdoc "devdoc" REMOVE` means that developer docs are moved to `$devdoc` *if* it exists and removed otherwise.

Reply:
@sternenseemann The fix is now in nixos-unstable. If you rebase, you no longer need the `fixupPhase`.